content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -345,7 +345,6 @@ SSH is required if you want to control all nodes from a single machine.
         scp /etc/kubernetes/pki/front-proxy-ca.crt "${USER}"@$host:
         scp /etc/kubernetes/pki/front-proxy-ca.key "${USER}"@$host:
         scp /etc/kubernetes/pki/etcd/ca.crt "${USER}"@$host:etcd-ca.crt
-        scp /etc/kubernetes/pki/etcd/ca.key "${USER}"@$host:etcd-ca.key
```

Comment:
we should do it like so, because this section covers both external and local etcd:

```
# Quote this line if you are using external CA
scp /etc/kubernetes/pki/etcd/ca.key "${USER}"@$host:etcd-ca.key
```